### DISCUSSION TITLE
[v5] Failed to connect to external mssql

### DISCUSSION THREAD

**Discussion by b23618** [adonisjs/core#2654] [Help]
Hello

I want connect external db from server another ip 

my code
.env
`MSSQL_SERVER=xx.xxx.xxx.xx\namesqlserver
MSSQL_PORT=1433
MSSQL_USER=sa
MSSQL_PASSWORD=xxxxxxxx
MSSQL_DB_NAME=dbname`

but error message 'Failed to connect to xx.xxx.xxx.xx\namesqlserver:1433 - getaddrinfo ENOTFOUND xx.xxx.xxx.xx\namesqlserver","code":"ESOCKET"}}'

**Comment by RomainLanz** [comment]
Hey @menphurk! 👋 

This does not seems related to AdonisJS.
Please, ensure you can connect from your host (where your AdonisJS application is running) to your database first.

**Comment by marcelodelta** [comment]
I have the same problem. Managed to solve?
